airflow/providers/greatexpectations/operators/greatexpectations.py

```diff
@@ -0,0 +1,130 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import logging
+
+from airflow.models import BaseOperator
+from airflow.exceptions import AirflowException
+from airflow.utils.decorators import apply_defaults
+import great_expectations as ge
+
+log = logging.getLogger(__name__)
+
+
+class GreatExpectationsOperator(BaseOperator):
+    @apply_defaults
+    def __init__(self,
+                 *,
+                 run_id=None,
```

Comment:
Update to the new run_name; that will then be paired with the run time automatically.

```suggestion
                 run_name=None,
```